compiler/src/openxla/compiler/nvgpu/test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The IREE Authors
```

Comment:
I think we flipped authors in this repo.

Reply:
Done.